portfolyo/__init__.py

```diff
@@ -17,6 +18,8 @@
 from .tools.tzone import force_agnostic, force_aware
 from .tools.unit import Q_, Unit, ureg
 from .tools.wavg import general as wavg
+from .tools2.concat import general as concat
```

Comment:
@Pizza2Pizza this line is already on line 9. But let's not stop the PR for this; just remove it from the plot_children branch after merging the new develop into it.